packages/legend-studio/src/stores/EditorStore.tsx

```diff
@@ -252,7 +252,7 @@ export class EditorStore {
       Boolean(
         this.sdlcState.currentProject &&
           this.sdlcState.currentWorkspace &&
-          this.sdlcState.currentRevision,
+          (this.sdlcState.currentRevision || this.isInViewerMode),
```

Comment:
I don't think this is the right place to do this, are you just trying to unblock hotkey here? If so, what you could do is to tackle `createGlobalHotKeyAction`